ci/build-helpers/README.md

```diff
@@ -0,0 +1,24 @@
+## Publish as a library
+This library exists to help to publish artifacts to Maven Central for CMP projects and its dependencies.
+
+For exmaple, for [Skiko](https://github.com/JetBrains/skiko/blob/29d227acb6893171940b120114ac1cbc95f50987/skiko/settings.gradle.kts#L19)
+```
+./gradlew publish
+```
+
+It is published automatically to https://maven.pkg.jetbrains.space/public/p/compose/internal on any change in the CMP sources by this CI task:
+https://teamcity.jetbrains.com/buildConfiguration/JetBrainsPublicProjects_Compose_PublishBuildHelpers
+
+## Publish as a local library
+```
+./gradlew publishToMavenLocal
+```
+
+## Use from sources
+```
+./gradlew -p=cli reuploadArtifactsToMavenCentral -Pmaven.central.sign=true \
```

Comment:
CI updated in https://jetbrains.team/p/ui/repositories/compose-teamcity-config/commits?query=head%3Arefs%2Fheads%2Figor.demin%2Fpublish-build-helpers&commits=3d60b217b7a2c6d2cab5a0b7c466c75223d9e6dc&tab=changes